.storybook/preview-head.html

```diff
@@ -1,9 +1,6 @@
 <link
-rel="stylesheet"
-href="//fonts.googleapis.com/css?family=Roboto:300,400,700,300italic|Inconsolata:400,700|Roboto+Condensed:400,700"
-type="text/css"
+  rel="stylesheet"
+  href="//fonts.googleapis.com/css?family=Roboto:300,400,700,300italic|Inconsolata:400,700|Roboto+Condensed:400,700"
+  type="text/css"
 />
 <script src="https://use.fortawesome.com/b5c7b73e.js"></script>
-
-<!-- Include modernizr. Requires Happa dev server to be running.-->
-<script type="text/javascript" src="http://localhost:7000/vendor/modernizr.js"></script>
```

Comment:
We only use this for the `download` attribute of the `a` element.

I believe we don't actually need it in storybook
https://caniuse.com/download

Reply:
https://github.com/giantswarm/happa/blob/72af9395d68fd4889fb22f29ca5050e9e8497242/src/components/UI/Display/Documentation/FileBlock.js#L102

Will need a bit of a change in order for it to work in the storybook then. Like to check if moderniser is there first.